otter_welcome_buddy/common/utils/discord_.py

```diff
@@ -0,0 +1,17 @@
+import logging
+
+import discord
+from discord.ext.commands import Context
+
+
+logger = logging.getLogger(__name__)
+
+
+async def send_plain_message(ctx: Context, message: str) -> None:
+    """Send a message as embed, this allows to use more markdown features"""
+    try:
+        await ctx.send(embed=discord.Embed(description=message, color=discord.Color.teal()))
+    except discord.Forbidden:
+        logger.exception("Not enough permissions to send the message")
```

Comment:
Instead of using logger.exception to log exceptions, consider using logger.error along with exc_info=True. This way, you provide the same exception information in the log but maintain more control over the log message. `    logger.error("Not enough permissions to send the message", exc_info=True)`